spec/models/project_accumulator_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe ProjectAccumulator, connect_to_mediaflux: true, type: :model do
+  let(:user) { FactoryBot.create(:user) }
+  let(:session_id) { user.mediaflux_session }
+  let(:project_in_mediaflux) { FactoryBot.create(:project_with_doi, status: Project::APPROVED_STATUS) }
+
+  describe "#create!" do
+    it "creates accumulators for the project" do
+      project_id = ProjectMediaflux.create!(session_id: user.mediaflux_session, project: project_in_mediaflux)
+      project_accumulators = described_class.new(project: project_in_mediaflux, session_id: session_id)
+      project_accumulators.create!
```

Comment:
What is the return value here.  Since the return value is significant below maybe it should be tested here